registry/repository.go

```diff
@@ -1,198 +1,117 @@
 package registry
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
 	"fmt"
 
-	"github.com/notaryproject/notation-go"
-	"github.com/opencontainers/go-digest"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
-	artifactspec "github.com/oras-project/artifacts-spec/specs-go/v1"
 	"oras.land/oras-go/v2"
 	"oras.land/oras-go/v2/content"
-	"oras.land/oras-go/v2/registry"
-	"oras.land/oras-go/v2/registry/remote"
+	orasRegistry "oras.land/oras-go/v2/registry"
```

Comment:
The alias `orasRegistry` is not necessary. Can we remove it?

It is OK to import a package with name `registry` in a package named `registry`. There is no name conflict.